doc/source/xml.rst

```diff
@@ -0,0 +1,22 @@
+==============
+ XML Samples
+==============
+
```

Comment:
It's pretty common to expect a JSON or YAML API to return a native type based on the input data, but for XML we're always going to return an ElementTree object. The examples in the test are great, but could you also add a note here saying that with a link to the python docs, so that users of the feature understand how they can potentially use the thing they're being given?

Reply:
I added a link but actually I am not surewhat else I can put in the docs.

Reply:
It's a start, at least. I mainly didn't want a consumer to have to read the code to find out what type of object they're receiving.